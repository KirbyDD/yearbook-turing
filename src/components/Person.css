.person {
  background-color: white;
  border: solid;
  box-sizing: border-box;
  margin: 5px;
  padding: 10px;
  width: 24%
}

img {
  width: 150px;
}