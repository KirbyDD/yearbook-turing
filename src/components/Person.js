import React from 'react';
import './Person.css';

const Person = (props) => {
  return (
    <article>
      <img src={props.photo} alt="Staff member"/>
      <h3>{props.name}</h3>
      <p>{props.quote}</p>
      <p>{props.superlative}</p>
    </article>
  )
}

export default Person;
