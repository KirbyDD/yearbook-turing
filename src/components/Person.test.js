import React from 'react';
import Person from './Person';
import { shallow } from 'enzyme'

describe('Person', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Person />)
    expect(wrapper).toMatchSnapshot();
  })
})