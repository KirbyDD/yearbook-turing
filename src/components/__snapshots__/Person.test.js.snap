// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person should match snapshot 1`] = `
<article
  className="person"
>
  <img
    alt="individual Person"
  />
  <h3 />
  <p />
  <p />
</article>
`;
